tests/model_serving/model_runtime/vllm_omni/utils.py

```
@@ -0,0 +1,265 @@
+        percentile: Percentile value in [0, 100] (e.g. 95.0 for p95).
+
+    Returns:
+        Interpolated percentile value.
+
+    Raises:
+        ValueError: If values is empty or percentile is out of range.
+    """
+    if not values:
+        raise ValueError("Cannot calculate percentile of empty list")
+    if not 0 <= percentile <= 100:
+        raise ValueError(f"Percentile must be 0–100, got {percentile}")
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)